services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/api_client/_public.py

```diff
@@ -443,6 +444,14 @@ async def configure_proxy(
         )
         await self._thin_client.proxy_config_load(proxy_endpoint, proxy_configuration)
 
+    async def get_service_inactivity(
+        self, dynamic_sidecar_endpoint: AnyHttpUrl
+    ) -> InactivityResponse:
+        response = await self._thin_client.get_containers_inactivity(
+            dynamic_sidecar_endpoint
+        )
+        return parse_obj_as(InactivityResponse, response.json())
```

Comment:
TIP: to parse in a pydantic model use `parse_obj` member
```python
InativityResponse.parse_obj(response.json())
```
to parse to a composition of models/types use `parse_obj_as` free funtion
```python
parse_obj_as(list[InativityResponse], some_list_of_jsons)
```

The main reason is that `parse_obj_as` creates always an intermediary root model that is unnecessary in the first case